docs/pipelines.md

```diff
@@ -255,6 +257,22 @@ is no output from `test-app`, so `build-app` uses `runAfter` to indicate that
 `test-app` should run before it, regardless of the order they appear in the
 spec.
 
+#### retries
+
+Sometimes is needed some policy for retrying tasks for various reasons such as network errors, missing dependencies or upload problems. 
+Any of those issue must be reflected as False (corev1.ConditionFalse) within the TaskRun Status Succeeded Condition. 
+For that reason there is an optional attribute called `retries` which declares how many times that task should be retried in case of failure,
```

Comment:
I really do not want to use the word *flakey* (or flaky) because known flakey tests are not good, and could be interpreted as we encourage people to have them.
About the technical information I think is a good idea to say it, I think this is a technical library to be used by developers. Can even help newbies to understand the relationships within Tekton, but is just my 2 cts.

 I do agree with the default value, is missing and should be explicit.

Reply:
Ah I see.  Yeah I totally understand wrt "flakey" - it's a loaded term.